handsontable/types/plugins/base/base.d.ts

```diff
@@ -17,6 +20,7 @@ export class BasePlugin {
   disablePlugin(): void;
   updatePlugin(): void;
   addHook(name: string, callback: () => void): void;
+  addHook<K extends keyof Events>(key: K, callback: Events[K] | Array<Events[K]>): void;
   removeHooks(name: string): void;
```

Comment:
```suggestion
  addHook<K extends keyof Events>(key: K, callback: Events[K] | Array<Events[K]>): void;
  removeHooks<K extends keyof Events>(name: K): void;
```
I think that the old typing for `addHook` can be replaced with that new one completely. WDYT to fix typing for the `removeHooks` method right away? Underneath the method uses the `removeHook` from the Core that that have [similar types](https://github.com/handsontable/handsontable/blob/master/handsontable/types/core.d.ts#L114).

Reply:
I wanted to replace it, idk why I left the old one :)